hack/build/build_image.sh

```diff
@@ -13,6 +13,8 @@ image=${1}
 tag=${2}
 debug=${3:-false}
 
+supported_architectures=("amd64" "arm64")
```

Comment:
if I want to build locally another architecture, since we support 4? May be it's rare case, but I think you allow it for Operator.

Reply:
made it configurable